components/SiteNavigation.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div>
+    <div
+      v-for="item in navigation"
+      :key="item.title"
+    >
+      <Disclosure
+        v-slot="{ open }"
+        as="div"
+        :default-open="currentPath.startsWith(item._path)"
+      >
+        <DisclosureButton
+          class="group mt-2 w-full flex items-center pl-2 pr-1 py-2 text-left text-content-primary text-base leading-6 font-medium rounded-md hover:bg-gray-200 focus:outline-none focus:ring-2 focus:ring-inset ring-focus-ring"
+          :class="currentPath.startsWith(item._path) ? 'bg-gray-100' : ''"
+        >
+          <component
+            :is="item.icon"
+            class="mr-3 h-6 w-6 flex-shrink-0 text-gray-400 group-hover:text-gray-500"
+            aria-hidden="true"
+          />
+          <span class="flex-1">{{ item.title }}</span>
+          <svg
```

Comment:
This is taken from the Tailwind template, it is the SVG for the small arrow when the disclosure expands.
I could pull it out into a separate file if preferred.

Reply:
@MeganSteenkamp Yep we should stick to the same convention for our icons 👍

Reply:
@tom-blake have updated this 👍